Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonPrintDialog.cs

```diff
@@ -62,15 +62,6 @@ public string Title
                 ClickCallback = ClickCallback
             };
 
-        /// <summary>
```

Comment:
Why is this removed

Reply:
It had a duplicate entry, so it wouldn't build